cgemController.py

```diff
@@ -50,39 +41,37 @@
     
         dec.deg = input ('decDeg : ')
         dec.min = input ('decMin : ')
+        
         dec.sec = input ('decSec : ')
 
         print 'ra  : ', ra.hr,   ra.min,  ra.sec
         print 'dec : ', dec.deg, dec.min, dec.sec
         print
 
         print 'r'+ra.toCgem()+','+dec.toCgem()
-
-        ser.write ('r'+ra.toCgem()+','+dec.toCgem())
+        # ser.write ('r'+ra.toCgem()+','+dec.toCgem()+'#')
+        # data = ser.read(50)
+        # print 'data : ', data
         
         # Hand controller should respond with a # character
 
-        # print 'Changed the timeout value to: ', ser.timeout
-        foundHashTag = True
-
-        while (foundHashTag):
-            data = ser.read(1)
-            print 'data : ', data
-            if (data == '#'):
-                print 'found the hash tag'
-                foundHashTag = False
-        print
-
-        ser.write(b'e')            # write a string
-        foundHashTag = True
-
-        output = ''
-
-        while (foundHashTag):
-            data = ser.read(1)
-            output = output + data
-            if (data == '#'):
-                foundHashTag = False
-        print 'output: ', output
-
-ser.close()
+# Leaving this present/commented for the moment - possible remove before merging
+#        foundHashTag = True
+#        while (foundHashTag):
+#            data = ser.read(1)
+#            print 'data : ', data
+#            if (data == '#'):
+#                print 'found the hash tag'
+#                foundHashTag = False
+#        ser.write(b'e')            # write a string
+#
+#        foundHashTag = True
+#        output = ''
+#        while (foundHashTag):
```

Comment:
@RobSteele49 I'm not clear on why this code block was duplicated - is this the pattern for getting an ACK from the telescope?  If so, it's a good candidate for a new `SerialInterface` object.